cardinal/ecs/gamestate/redis.go

```diff
@@ -13,10 +13,121 @@ import (
 	"pkg.world.dev/world-engine/cardinal/types/component"
 )
 
+type RedisStorage struct {
+	OtherClient   redis.Cmdable
+	CurrentClient redis.Cmdable
```

Comment:
these should be made private fields

should also comment what the differentiation of the clients is for. not sure if OtherClient is the best word either. if we could do something like `txClient` and `client` that might be a bit more clear

Reply:
Basically it swaps the transaction and the client back and forth as you enter transactions. cachedClient isn't appropriate either imo. I'll think of something.

Reply:
Actually you know what, you're right, there's no other name I can think of. Let's go with cached